cfgov/cfgov/settings/test.py

```diff
@@ -92,3 +92,6 @@
 HAYSTACK_CONNECTIONS["default"]["INDEX_NAME"] = (
     "test_" + HAYSTACK_CONNECTIONS["default"]["INDEX_NAME"]
 )
+
+ELASTICSEARCH_DSL_AUTO_REFRESH = False
+ELASTICSEARCH_DSL_AUTOSYNC = False
```

Comment:
```suggestion
ELASTICSEARCH_DSL_AUTOSYNC = False

```

We exclude settings from linting, but I'd spend a line-ender.